app/main.py

```diff
@@ -66,67 +66,72 @@ def upload():
     csv = request.form['csv'].strip()
     img = request.form['img-default']
     eventyay_url = request.form['eventyay_url'].strip()
+    text_on_image = request.form['text_on_image']
     file = request.files['file']
     _pdf = True if request.form.get('pdf') == 'on' else False
     _zip = True if request.form.get('zip') == 'on' else False
 
     if not _pdf and not _zip:
         flash('Please select a download filetype!', 'error')
         return redirect(url_for('index'))
-    if file.filename == '' and csv == '':
-            flash('Please select a CSV file to Upload!', 'error')
-            return redirect(url_for('index'))
 
     # If default background is selected
     if img != '':
         if(img == 'user_defined.png'):
             bg_color = request.form['bg_color']
             text_on_image = request.form['text_on_image']
             do_svg2png(img, 1, bg_color, text_on_image)
+            filename = img + '.csv'
```

Comment:
What is the point of adding this line here? The `filename = img + '.csv'` is going to execute in the next line anyway

Reply:
sorry i did not actually see that one. will remove it